src/mame/drivers/galaxold.cpp

```diff
@@ -2383,13 +2370,6 @@ MACHINE_CONFIG_START(galaxold_state::scrambler)
 	/* basic machine hardware */
 	MCFG_DEVICE_MODIFY("maincpu")
 	MCFG_DEVICE_PROGRAM_MAP(scrambler_map)
-
-	/* video hardware */
-	MCFG_PALETTE_MODIFY("palette")
-	MCFG_PALETTE_ENTRIES(32+2+64+1)  /* 32 for the characters, 2 for the bullets, 64 for the stars, 1 for background */
-
-	MCFG_PALETTE_INIT_OWNER(galaxold_state,scrambold)
-	MCFG_VIDEO_START_OVERRIDE(galaxold_state,scrambold)
```

Comment:
What's the actual reason for removing these?  It's not clear from the description.